R/config.R

```diff
@@ -37,7 +37,7 @@ get_tools_project <- function(tool_name) {
 #' @export
 get_conf <- function(project_name) {
 
-  conf_path <- paste0("../conf/", project_name, ".yml")
+  conf_path <- paste0("conf/", project_name, ".yml")
```

Comment:
I am not sure if this function makes much sense. You will still need to hardcode the project name to the notebook, which makes sense: The notebook should work for any config file, and the user has to select one.

Remember what we want to encapsulate to prevent cascading effect is the contents of the config file itself. The path to find them is less a concern, and we certainly do not want to force the user on only allowing to choose in one place. It goes against the philosophy of letting the user type the paths of all files in the config file.

Just accept the project_name default parameter as the full path. The default value will serve as documentation for the user.